### PR TITLE
Fix/simplify validation

### PR DESCRIPTION
**Pull request by codeflyer**
In thi PR:

- Update the validation function, removing not necessary check.
- Rename check function to `checkForCompleteDag`

### CODE REVIEW

**Review by olizilla — APPROVED** [review]
👍

File: validator/lib/processCars.js [hunk 1]
```diff
@@ -35,10 +34,14 @@ export async function validateCar (record, { s3, destinationBucket }) {
...
+    const { structure, blocksIndexed } = await checkForCompleteDag(s3Object.Body)
 
-    if (validationCarResult.structure !== 'Complete') {
-      throw new Error(`Structure not complete: ${validationCarResult.structure}, size: ${validationCarResult.size}, blocks: ${validationCarResult.blocks}`)
+    if (blocksIndexed === 0) {
+      throw new Error('empty CAR, zero blocks found')
+    }
+
+    if (structure !== 'Complete') {
+      throw new Error(`Structure not complete: ${structure}, blocks: ${blocksIndexed}`)
```
Comment (olizilla) [inline validator/lib/processCars.js:44]: Would be helpful to get the complete linkdex report output here

**Comment by olizilla** [comment]
Note that as we are just iterating the blocks from the response stream, this process doesn't really need any disk space beyond what it takes to run the node script.

**Comment by codeflyer** [comment]
> Note that as we are just iterating the blocks from the response stream, this process doesn't really need any disk space beyond what it takes to run the node script... i'm looking at the fargate config here
> 
> https://github.com/web3-storage/pickup/blob/5e401f1c0ab81d8326e4d35a356dc0ca729afec2/stacks/PickupStack.ts#L220-L222
> 
> Do we have graphs that show the validation process resource usage?

I did some test in the previous process. I'll do something to check the usage here.

**Pull request merged**